src/pytest/test_replicate.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import os
+import os.path
+from RLTest import Env
+import time
+
+def testDelReplicate():
+  env = Env(useSlaves=True)
+  master = env.getConnection()
+  slave = env.getSlaveConnection()  
+  env.assertContains("PONG", master.execute_command("ping"))
+  env.assertContains("PONG", slave.execute_command("ping"))  
+
+  env.assertOk(master.execute_command('ft.create', 'idx', 'schema', 'f', 'text'))
+
+  for i in range(10):
+    master.execute_command('ft.add', 'idx', 'doc%d' % i, 1.0, 'fields',
+                                      'f', 'hello world')
+
+  time.sleep(5)
```

Comment:
no sleep in tests, please.

Reply:
You might be able to do something like a timed poll..

Reply:
another thing you can do is make a 'sentinel' key. that is, create a key at the beginning of the test, then delete it at the end. because commands are executed in order, it would be safe to assume that if the key is missing at the slave, all prior operations to the last DEL (not FT.DEL but simple DEL) have been replicated. This provides a reliable method of synchronization.

Reply:
@mnunberg 
Implemented using TimeLimit